src/sentry/integrations/slack/actions/notification.py

```diff
@@ -154,56 +160,108 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                     # To reply to a thread, use the specific key in the payload as referenced by the docs
                     # https://api.slack.com/methods/chat.postMessage#arg_thread_ts
                     payload["thread_ts"] = parent_notification_message.message_identifier
+                    thread_ts = parent_notification_message.message_identifier
                     # If this flow is triggered again for the same issue, we want it to be seen in the main channel
                     payload["reply_broadcast"] = True
+                    reply_broadcast = True
+
+            organization = event.group.project.organization
+
+            if organization.id in options.get("slack.sdk-issue-alert"):
```

Comment:
might be good to just pull this out into it's own function, what do you think? Will make it easier to maintain and switch as well when the migration is done

Reply:
good idea